rdf-toolkit.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# uncomment (remove "#") the next line to download "rdf-toolkit.jar" to the "${HOME}" directory
+#export JARP="${HOME}"
+
+export JARURL=https://jenkins.edmcouncil.org/view/rdf-toolkit/job/rdf-toolkit-build/lastSuccessfulBuild/artifact/target/
+
+# === don't touch anything below
+
+test -n "${1}" || { echo "ERR: missing arg - full path to .ttl file" ; read -n 1 -s -r -p "Press any key to continue...." ; exit 1 ; }
+
+curl --version &>/dev/null || { echo "ERR: missing curl" ; read -n 1 -s -r -p "Press any key to continue...." ; exit 1 ; }
+
+java --version &>/dev/null || { echo "ERR: missing java - download e.g. from https://openjdk.org/install/ and install" ; read -n 1 -s -r -p "Press any key to continue...." ; exit 1 ; }
+
+test -n "${JARP}" || export JARP="${TMP:-/tmp}"
```

Comment:
As we discussed to align script to macos please change TMP to TMPDIR in all places

Reply:
Resolved in commit 0bf817b